.github/workflows/development-environment.yml

```diff
@@ -65,8 +65,7 @@ jobs:
 
       - name: Set up development environment (as per docs)
```

Comment:
Reminder to update the relevant docs.

Reply:
I landed the change yesterday: https://develop.sentry.dev/environment/#virtual-environment

Reply:
No, but I see the PR - will approve. https://github.com/getsentry/develop/pull/237